src/onboarding/welcome/Welcome.tsx

```diff
@@ -54,15 +57,25 @@ export default function Welcome() {
       <View style={{ marginBottom: Math.max(0, 40 - insets.bottom) }}>
         <Button
           onPress={onPressCreateAccount}
-          text={t('welcome.createAccount')}
+          text={
+            createAccountCopyTestConfig === 'control'
+              ? t('welcome.createAccount')
+              : t('welcome.createNewWallet')
+          }
           size={BtnSizes.FULL}
           type={BtnTypes.ONBOARDING}
           style={styles.createAccountButton}
           testID={'CreateAccountButton'}
         />
         <Button
           onPress={onPressRestoreAccount}
-          text={t('welcome.restoreAccount')}
+          text={
+            createAccountCopyTestConfig === 'control'
+              ? t('welcome.restoreAccount')
+              : createAccountCopyTestConfig === 'treatment1'
```

Comment:
i know it's a little arbitrary what each variant is doing, but i still think it will be easier to understand if we try to give each variant a name. like "account" / "wallet" / "alreadyHaveWallet"

Reply:
I did as you said!